src/snowflake/snowpark/modin/plugin/extensions/general_overrides.py

```diff
@@ -101,513 +101,179 @@
 ]
 
 
+###########################################################################
+# Data manipulations
+###########################################################################
+
+
 @snowpark_pandas_telemetry_standalone_function_decorator
-@_inherit_docstrings(pandas.isna, apilink="pandas.isna")
-def isna(obj):  # noqa: PR01, RT01, D200
-    """
-    Detect missing values for an array-like object.
+def melt(
```

Comment:
are those changes just move those methods around and no actual code changes in the method?

Reply:
Yes, there is actual code change here.